frontend/lib/onboarding/onboarding-step-4.tsx

```diff
@@ -22,13 +22,41 @@ import {
   OnboardingStep4Version2Mutation,
 } from "../queries/OnboardingStep4Version2Mutation";
 import { trackSignup } from "../analytics/track-signup";
+import { Link } from "react-router-dom";
+import JustfixRoutes from "../justfix-route-info";
+import { useAutoFocus } from "../ui/use-auto-focus";
 
 type OnboardingStep4Props = {
   routes: OnboardingRouteInfo;
   toSuccess: string;
   signupIntent: OnboardingInfoSignupIntent;
 };
 
+const ExistingAccountNotification: React.FC<{}> = () => {
+  const ref = useRef<HTMLDivElement | null>(null);
+
+  useAutoFocus(ref, true);
```

Comment:
This will ensure that users don't miss the notification, either because they're on a screen reader or because they are otherwise scrolled too far down on the screen to see the alert.